Plugins/Sycl/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+# This module needs at least CMake 3.13.
+cmake_minimum_required( VERSION 3.13 )
+
+find_package(SYCL REQUIRED)
+
+add_library(
+  ActsPluginSycl SHARED
+  # header files
+  include/Acts/Plugins/Sycl/Seeding/detail/Types.h
```

Comment:
:confused: Did you leave this as `Types.h` on purpose, or was this just an oversight?